app/services/idempotence/validator.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'redis'
+
+module Idempotence
+  class Validator
+    TTL = 43200 # 12 hours in seconds
+
+    def call(key, required: true)
```

Comment:
what is the format of the keys we are going to be accepting? We might need to add validation for those

Reply:
In the swagger docs I have specified it should be a uuid - I'll add a validation for that 👍